typescript-generator-core/src/main/java/cz/habarta/typescript/generator/Settings.java

```diff
@@ -140,6 +143,13 @@ public void loadExtensions(ClassLoader classLoader, List<String> extensions, Lis
         }
     }
 
+    public static List<Class<? extends Annotation>> loadAnnotations(ClassLoader classLoader, List<String> stringAnnotations) {
```

Comment:
could you please replace this method with `loadNonConstEnumAnnotations` which would be consistent with other `load*` methods?

Reply:
This one would actually now be more generic, since it returns those annotations instead of setting them on a specific field. This could replace some of the others actually

Reply:
ok, leave your method as it is now and I will use it on other places too, or maybe I will change `loadClasses` to return empty list instead of `null` and use it directly